Makefile

```diff
@@ -0,0 +1,5 @@
+build:
+	sed -e "s/\$${UNISON_VERSION}/2.48.3/"   -e "s/\$${OCAML_VERSION}/4.02/" Dockerfile > Dockerfile-unison2.48.3-ocaml4.02
+	sed -e "s/\$${UNISON_VERSION}/2.48.3/"   -e "s/\$${OCAML_VERSION}/4.01/" Dockerfile > Dockerfile-unison2.48.3-ocaml4.01
+	sed -e "s/\$${UNISON_VERSION}/2.40.102/" -e "s/\$${OCAML_VERSION}/4.02/" Dockerfile > Dockerfile-unison2.40.102-ocaml4.02
+	sed -e "s/\$${UNISON_VERSION}/2.40.102/" -e "s/\$${OCAML_VERSION}/4.01/" Dockerfile > Dockerfile-unison2.40.102-ocaml4.01
```

Comment:
I do agree. I didn't test it so I am sure there is likely an error somewhere. I can open a PR for it later once this is done. We have rewritten the entire project in these two PR's at this point.